tensorflow_datasets/core/holder.py

```diff
@@ -0,0 +1,295 @@
+import os
+import re
+import tarfile
+
+import tensorflow as tf
+from tensorflow_datasets.core.utils import py_utils
+from tensorflow_datasets.core import lazy_imports
+from tensorflow_datasets.core import naming
+from tensorflow_datasets.scripts import create_new_dataset
+import json
+import shutil
+import zipfile
+
+
+# TODO if download 2 files return array and download both
+
+NUMBER_OF_CREATED_FILE = 0
+
+class Holder(object):
+
+	def __init__(self, name, file_type, path, output_path=None):
+		self.name = name  # image
+		self.typ = file_type  # png
+		self.path = path  # /folder/image.png
+		self.output_path = output_path  # /target_folder/image.png
+		self.create_basedir()
+
+	def create_basedir(self):
+		basedir = os.path.dirname(self.output_path)
+		if not tf.io.gfile.exists(basedir):
+			tf.io.gfile.makedirs(basedir)
+
+
+class ImageHolder(Holder):
+
+	def __init__(self, zip_file=None, *args, **kwargs):
+		super(ImageHolder, self).__init__(*args, **kwargs)
+		self.zip_file = zip_file
```

Comment:
I think indentation is using tabs here.
Please use 2 spaces for each level of indentation, to be coherent with the rest of the project.

Reply:
Sorry I missed it.